### PR TITLE
Fix docker-compose.yml

### PR DESCRIPTION
**Pull request by yamadayuki**
## WHY

For https://github.com/wantedly/sync-messenger-rails/pull/2354
When I build kong-frontend and kong-token-registrator, I send `POST` request to create access_token, I can make the consumer but I receive an error.
## WHAT

Fix `docker-compose.yml` properly.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,16 +1,5 @@
 version: '2'
 services:
-  kong-frontend:
```
Comment (koudaiii) [inline docker-compose.yml:3]: mistake commit? 👀 
I think that has nothing to do with this pull request 👀

Comment (yamadayuki) [reply docker-compose.yml:3]: Ah yes. It is a mistake. I'll fix it and then ping you.

Comment (yamadayuki) [reply docker-compose.yml:3]: @koudaiii Fixed this and I've confirmed in my local env.

1 docker-compose up docker-compose 2016-09-21 21-36-01

Comment (koudaiii) [reply docker-compose.yml:3]: Thnak you 👍

**Comment by koudaiii** [comment]
LGTM!! 👍

**Comment by yamadayuki** [comment]
Thank you 🐻

**Pull request merged**